app/lib/server/methods/deleteMessage.js

```diff
@@ -28,7 +28,11 @@ Meteor.methods({
 			},
 		});
 
-		if (!originalMessage || !canDeleteMessage(uid, originalMessage)) {
+		if (!originalMessage) {
+			return;
+		}
+
+		if (!canDeleteMessage(uid, originalMessage)) {
```

Comment:
@shubhsherl  Is this change needed? In upstream it is not there

Reply:
So if you delete any offline message, if the app is connected (like attachment, which is still uploading) and you happen to delete it. then you get a toast showing the error, which is unnecessary.

Reply:
Okay sounds good. Can you add same comments in the code? For future reference Thanks 😄